knowledge-base/insert-html-content-into-pdf-tablecell-radpdfprocessing.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Inserting HTML Content into PDF TableCell with RadPdfProcessing
+description: Learn how to insert HTML content into a TableCell in a PDF document using RadPdfProcessing and RadWordsProcessing libraries.
+type: how-to
+page_title: How to Insert HTML Content into PDF TableCell Using RadPdfProcessing
+slug: insert-html-content-into-pdf-tablecell-radpdfprocessing
+tags: pdfprocessing, document, processing, pdf, html, table, cell, insert, wordsprocessing
+res_type: kb
+ticketid: 1671595
+---
+
+## Environment 
+
+| Version | Product | Author | 
+| --- | --- | ---- | 
+| 2024.3.806 .NET Standard| RadWordsProcessing-RadPdfProcessing |[Desislava Yordanova](https://www.telerik.com/blogs/author/desislava-yordanova)| 
+
+## Description
+When generating PDF documents, a common requirement is to insert HTML content into specific sections of the document, such as a [TableCell]({%slug radpdfprocessing-editing-tablecell%}). This article demonstrates how to achieve this using the smooth integration between [RadPdfProcessing]({%slug radpdfprocessing-overview%}) and [RadWordsProcessing]({%slug radwordsprocessing-overview%}) libraries. 
+
+>caption Sample HTML content to Insert
+
+```HTML
+<!DOCTYPE html>
+<html>
+<body>
+<p>I am normal</p>
+<p style="color:red;">I am red</p>
+<p style="color:blue;">I am blue</p>
+<p style="font-size:50px;">I am big</p>
+</body>
+</html>
+```
+
+![HTML Displayed in the Browser](images/sample-html-content.png)  
+
+## Solution
+To insert HTML content into a `TableCell` in a PDF document, you can obtain the HTML content as an image and insert the image inside the PDF table cell. Below are the steps and complete code snippet for achieving this:
+
+1. **Import HTML Content**: Use the RadWordsProcessing's [HtmlFormatProvider]({%slug radwordsprocessing-formats-and-conversion-html-htmlformatprovider%}) to import the HTML content into a [RadFlowDocument]({%slug radwordsprocessing-model-radflowdocument%}).
+
+1. **Export the HTML Content to PDF Format**: Use the RadWordsProcessing's [PdfFormatProvider]({%slug radwordsprocessing-formats-and-conversion-pdf-pdfformatprovider%}) to convert the `RadFlowDocument` into PDF format.
+
+1. **Convert the exported PDF content to an Image**: Use the RadPdfProcessing's [PdfFormatProvider]({%slug radpdfprocessing-formats-and-conversion-pdf-pdfformatprovider%}) to import the exported HTML content to [RadFixedDocument]({%slug radpdfprocessing-model-radfixeddocument%}) and the [SkiaImageFormatProvider]({%slug radpdfprocessing-formats-and-conversion-image-using-skiaimageformatprovider%}) to export the PDF pages to images.
```

Comment:
"import the exported HTML content" can cause misunderstandings. An actual HTML content no longer exists, it is now converted to PDF so maybe add a clarification that this is the PDF-converted HTML content.